sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -379,19 +378,16 @@ static void _netif_list_ipv6(ipv6_addr_t *addr, uint8_t flags)
                 break;
         }
     }
-    line_thresh = _newline(0U, line_thresh);
+    puts("");
 }
 
 static void _netif_list_groups(ipv6_addr_t *addr)
 {
-    unsigned line_thresh = _LINE_THRESHOLD;
-
     if ((ipv6_addr_is_multicast(addr))) {
         char addr_str[IPV6_ADDR_MAX_STR_LEN];
         ipv6_addr_to_str(addr_str, addr, sizeof(addr_str));
-        printf("inet6 group: %s", addr_str);
+        printf("inet6 group: %s\n", addr_str);
```

Comment:
This changes from always printing  `\n (many spaces) ` to sometime only printing a new line.
Not sure if it is wanted.

Reply:
Well the line is always too long and int should be broken after one group in any case.